src/main/java/io/github/tryexceptelse/jdex/be/Rolodex.java

```diff
@@ -26,7 +22,8 @@
      * Default constructor for Rolodex.
      * Should use default file location
      */
-    public Rolodex() {
+    public Rolodex()
+    {
         // skeleton placeholder.
```

Comment:
Constructors of Rolodex should initialize the contacts ArrayList or the other methods will have nothing to operate on